Remotion/Validation/UnitTests/RoleCollectors/RemovingObjectValidatorRegistrationTest.cs

```diff
@@ -29,7 +30,7 @@ public class RemovingObjectValidatorRegistrationTest
     [Test]
     public void Initialization ()
     {
-      var validatorType = new Mock<IObjectValidator>().GetType();
+      var validatorType = typeof(FakeCustomerValidator);
```

Comment:
Instead of your change, you could also do `new Mock<IObjectValidator>().Object.GetType();` This would be closer to the original code. Using a random fake type is also okay.